lib/engine/game/g_18_eus/step/buy_train.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/buy_train'
+
+module Engine
+  module Game
+    module G18EUS
+      module Step
+        class BuyTrain < Engine::Step::BuyTrain
+          def actions(entity)
```

Comment:
Will need to merge in my "skip_bny" include

Reply:
Yes. I'll handle after your PR comes in.